app/src/main/java/com/tari/android/wallet/model/TariWalletAddress.kt

```diff
@@ -53,27 +55,12 @@ data class TariWalletAddress(val hexString: String = "", val emojiId: String = "
     override fun toString(): String = "TariWalletAddress(hexString='$hexString', emojiId='$emojiId')"
 
     companion object {
-        const val HEX_ZERO = "0000000000000000000000000000000000000000000000000000000000000026"
-        const val HEX_ZERO_66 = "000000000000000000000000000000000000000000000000000000000000000026"
-        const val EMOJI_ZERO =
+        private const val HEX_ZERO = "0000000000000000000000000000000000000000000000000000000000000026"
+        private const val HEX_ZERO_66 = "000000000000000000000000000000000000000000000000000000000000000026"
+        private const val EMOJI_ZERO =
             "\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF00\uD83C\uDF57"
 
-        // TODO I don't like this
-        fun createWalletAddress(hexString: String = HEX_ZERO, emojiId: String = ""): TariWalletAddress {
-            // crunch fix for not crashing on action related to wallet address
-            return if (hexString == HEX_ZERO) {
-                TariWalletAddress(
-                    hexString = HEX_ZERO_66,
-                    emojiId = EMOJI_ZERO,
-                )
-            } else {
-                TariWalletAddress(
-                    hexString = hexString,
-                    emojiId = emojiId,
-                )
-            }
-        }
-
-        fun validate(addressHex: String): Boolean = addressHex.length > 64
+        // Empty wallet address for cases such one-sided payment or phone contact
```

Comment:
Are we still need those "zeroed" values?

Reply:
Zeroed wallet is the wallet from a phone contact or from one-sided payment. It's just a default empty value

Reply:
Can't you use optionals instead?

Reply:
We can discuss it on the meeting. In the actual version we show default address (zeroed) if there no address in a contact detail